vectors/ecobee/ecobee.svg

```diff
@@ -0,0 +1,5 @@
+<svg version="1.1" id="logo-text" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" style="enable-background:new 0 0 101.58 21.57;" xml:space="preserve" viewBox="Infinity Infinity -Infinity -Infinity">
```

Comment:
It looks like this logo is not responsive yet. So it won't show up in Raivo OTP correctly.

Reply:
The rest of the icons are okay.

Reply:
I've deleted ecobee for now. :)